cmd/complytime/cli/info.go

```diff
@@ -0,0 +1,644 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+	"io"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	oscalTypes "github.com/defenseunicorns/go-oscal/src/types/oscal-1-1-3"
+	"github.com/oscal-compass/oscal-sdk-go/models/components"
+	"github.com/oscal-compass/oscal-sdk-go/settings"
+	"github.com/oscal-compass/oscal-sdk-go/validation"
+
+	"github.com/spf13/cobra"
+
+	"github.com/complytime/complytime/cmd/complytime/option"
+	"github.com/complytime/complytime/internal/complytime"
+	"github.com/complytime/complytime/internal/terminal"
+)
+
+const (
+	colWidthControlID      = 12
+	colWidthControlTitle   = 40
+	colWidthImplStatus     = 21
+	colWidthPluginsUsed    = 24
+	colWidthRulesInControl = 55
+	colWidthPluginUsed     = 31
+)
+
+var (
+	tableHeaderStyle = table.DefaultStyles().Header.
+				BorderStyle(lipgloss.NormalBorder()).
+				BorderForeground(lipgloss.Color("240")). // Light grey border
+				BorderBottom(true).
+				Bold(false)
+
+	tableCellStyle = table.DefaultStyles().Cell.
+			Foreground(lipgloss.Color("250")). // Very light grey text
+			Bold(true)
+
+	// Style for key-value pair keys
+	keyStyle = lipgloss.NewStyle().
+			PaddingRight(1)
+
+	// Style for key-value pairs values
+	valueStyle = lipgloss.NewStyle().
+			Foreground(lipgloss.Color("250")).
+			Bold(true)
+
+	// Container style for key-value information blocks
+	infoContainerStyle = lipgloss.NewStyle().
+				Border(lipgloss.NormalBorder()).
+				BorderForeground(lipgloss.Color("240")). // Light grey border for containers
+				Padding(0, 1).
+				Width(90) // Fixed width for consistent formatting
+)
+
+// Rule represents details about a rule for easy
+// mapping of rules to plugins.
+type Rule struct {
+	ID          string
+	Plugin      string
+	Description string
+	Parameters  []string
+}
+
+// Control repsents details about a control across
+// component sources.
+type Control struct {
+	ID                  string
+	Title               string
+	Description         string
+	ImplemenationStatus string
```

Comment:
```suggestion
	ImplementationStatus string
```
Unfortunately the typo was propagated. Could you update it, please?